games-fps/quakespasm/quakespasm-0.96.1.recipe

```diff
@@ -25,9 +25,9 @@ COPYRIGHT="1996-2001 Id Software, Inc.
 	2002-2009 John Fitzgibbons and others
 	2010-2014 QuakeSpasm developers"
 LICENSE="GNU GPL v2"
-REVISION="1"
+REVISION="3"
```

Comment:
Revision should be reset to 1 when updating the package to a new version.

Reply:
ah, apologies